package netserver;

//用户信息
message GetUserNameRequest
{
	required int32 userId = 1;
}

message GetUserNameResponse
{
	required int32  gender = 1;    // 性别
	required string name = 2;      // 名字
	optional string province = 3;  // 省份
}

message SetUserNameRequest
{
	required int32 gender = 1;  
	required string name = 2;
	optional string province = 3;
}